python/rascaline/log_utils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+# enums as integers https://docs.rs/log/0.4.0/log/enum.LevelFilter.html
+RUST_LOG_LEVEL_OFF = 0
+RUST_LOG_LEVEL_ERROR = 1
+RUST_LOG_LEVEL_WARN = 2
+RUST_LOG_LEVEL_INFO = 3
+RUST_LOG_LEVEL_DEBUG = 4
+RUST_LOG_LEVEL_TRACE = 5
+
+DEFAULT_LOG_LEVEL = RUST_LOG_LEVEL_WARN
+
+logging.basicConfig(filename='rascaline.log', level=DEFAULT_LOG_LEVEL)
```

Comment:
I don't think we want to write to files by default. If the end user want to do this, they can; but I would not pollute the filesystem if possible.